static/lib/jquery.cookie.js

```diff
@@ -6,109 +6,112 @@
  * Released under the MIT license
```

Comment:
Have you checked for newer version?

Reply:
We have the [v1.4.1](https://github.com/carhartl/jquery-cookie/releases/tag/v1.4.1) release on 3 Apr 2017 for jQuery Cookie Plugin, his last release was 1.4.1 because reading the README file said "This project was moved to https://github.com/js-cookie/js-cookie... View documentation for the latest backwards compatible release (1.5.1)". then checking the [README for js-cookie v1.5.1](https://github.com/js-cookie/js-cookie/tree/v1.5.1) said "Migrating from jquery-cookie", is a guite to migrate from jquery-cookie to JavaScript Cookie

Reply:
Done with commit [f3cf488](https://github.com/macagua/website_shops_map/commit/f3cf488a9db6727507ef4b6ef6e44137583d6017)